test/relayer_gaia_test.go

```diff
@@ -58,6 +58,9 @@ func TestGaiaToGaiaStreamingRelayer(t *testing.T) {
 	rlyDone, err := relayer.RunStrategy(src, dst, path.MustGetStrategy(), path.Ordered())
 	require.NoError(t, err)
 
+	// Wait for relay message inclusion in both chains
+	require.NoError(t, src.WaitForNBlocks(1))
```

Comment:
there is some interesting race conditions happening in `RunStrategy`. I think it needs to be looked over some more because I think it causes update client to fail